lib/flex_commerce_api/api_base.rb

```diff
@@ -97,9 +97,15 @@ def public_attributes
       attributes.reject { |k, v| PRIVATE_ATTRIBUTES.include?(k.to_s) }
     end
 
+    def meta_attribute(key)
+      attributes[:meta_attributes][key][:value] rescue nil
+    end
+
     def method_missing(method, *args)
       if relationships and relationships.has_attribute?(method)
         super
+      elsif meta_attribute(method)
+        meta_attribute(method)
```

Comment:
Not sure I agree with doing this here.  We've spent time extracting the meta attributes into their own bit of JSON only to flatten the structure again in the gem?